cypress/integration/journeys/manage-a-help-request.spec.js

```diff
@@ -9,6 +9,7 @@ describe;
 context('When you view a helpcase profile', () => {
     it('it allows you to naviagte to the manage request page', () => {
         cy.url().should('match', /\/helpcase-profile\/\d+$/);
+        cy.wait(1000)
```

Comment:
Not sure, but don't think this is necessary

Reply:
nope, removed